sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/dashboard/Surveillance/SurveillanceDashboardPage.java

```diff
@@ -32,4 +32,9 @@
   public static final By CASE_COUNTER =
       By.cssSelector("[location='case'] > div > div > div > div > div");
   public static final By TAB_SHEET_CAPTION = By.cssSelector(".v-tabsheet > div > div > div > div");
+  public static final By TIME_PERIOD_COMBOBOX = By.id("currentPeriod");
+  public static final By TIME_PERIOD_YESTERDAY_BUTTON = By.id("dashboardYesterday");
+  public static final By REFERENCE_DEFINITION_FULFILLED_CASES_NUMBER =
+      By.xpath(
+          "/html/body/div[1]/div/div[2]/div/div[2]/div/div/div/div[2]/div/div/div[3]/div/div[2]/div/div/div[2]/div/div/div/div[7]/div/div[2]/div");
```

Comment:
can we change this locator? it will fail once any dev will change a FE component from this chain

Reply:
It's one of several identical fields without any unique ID, so there's no prettier way of locating it that I know of.

Reply:
ok, we'll have to ask a dev after to help out adding an identifier there